### PR TITLE
Change scenario definitions to be usable instead of templates

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/132335

Mergeable after deployment of https://github.com/os-autoinst/openQA/pull/5277

### CODE REVIEW

File: scenario-definitions.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ machines:
       HDDSIZEGB: "20"
       QEMUCPU: host
       QEMURAM: "2048"
+      MODE: install''
```
Comment (Martchus) [inline scenario-definitions.yaml:16]: The duplicated `''` looks wrong.

**Comment by perlpunk** [comment]
The new scenario would now be run for every pull request, right? do we want that?

File: scenario-definitions.yaml [hunk 2]
```diff
@@ -13,6 +13,8 @@ machines:
       HDDSIZEGB: "20"
       QEMUCPU: host
       QEMURAM: "2048"
+      MODE: install
+      INSTALL_ONLY: '1'
```
Comment (perlpunk) [inline scenario-definitions.yaml:17]: These vars aren't used anymore in the code

**Review by perlpunk — CHANGES_REQUESTED** [review]
See comment

**Comment by mimi1vx** [comment]
> The new scenario would now be run for every pull request, right? do we want that?

this is a good question :D I think is little overkill on other side it will use almost all code paths in tests so shows almost all possible problems early

File: scenario-definitions.yaml (hunk 2 above)
Comment (mimi1vx) [reply scenario-definitions.yaml:17]: thx,

**Review by kalikiana — APPROVED** [review]
Looks good. Imho we can look into further splitting later if necessary.

**Review by perlpunk — APPROVED** [review]

**Pull request merged**